internal/xds/translator/cluster.go

```diff
@@ -225,42 +237,80 @@ func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
 	// Set Load Balancer policy
 	//nolint:gocritic
 	if args.loadBalancer == nil {
-		cluster.LbPolicy = clusterv3.Cluster_LEAST_REQUEST
+		leastRequest := &least_requestv3.LeastRequest{
```

Comment:
these changes are affecting xds config to change for the default case, curious why is this needed in this PR ?

Reply:
My zone aware changes were added to LocalityLbConfig vs the legacy CommonLbConfig. We don't necessarily need to remove `cluster.LbPolicy` but `cluster.LoadBalancingPolicy` will supersede any value set here so thought I'd just clean up unused fields.

Reply:
Moved this logic into a separate, temporary func as discussed and can followup with a change for making `cluster.LoadBalancingPolicy` the default